source/ui/REIXS/REIXSRIXSScanConfigurationView.cpp

```diff
@@ -232,8 +233,13 @@ REIXSRIXSScanConfigurationView::REIXSRIXSScanConfigurationView(REIXSXESScanConfi
 	RIXSView->addWidget(XASEnergySelect,0,1,3,1);
 	RIXSView->addWidget(queueView_, 3,0,1,2);
 
+	QVBoxLayout* outerVLayout = new QVBoxLayout();
```

Comment:
This is okay, but it would have been much easier to use the `AMScanConfigurationViewHolder3` constructor that takes the configuration view, a title, and an icon name and it builds the top frame for you.